pipeline-scripts/buildlib.groovy

```diff
@@ -1254,4 +1275,35 @@ def assertBuildPermitted(doozerOpts) {
     }
 }
 
+/**
+ * Run elliott find-builds and attach to given advisory, if not running a DRY_RUN.
```

Comment:
The method doesn't actually look at `DRY_RUN` so I'd omit mention of that.

Reply:
:heavy_check_mark: done